### PR TITLE
phpversion - updated support of not EOL versions

### PR DESCRIPTION
**Pull request by jwderoos**
| Q             | A
| ------------- | ---
| Branch        | v2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Documented?   | yes

phpversion errors on php 8.1 as original support was set to end on the 25th of november 2024. Official support was extended. Also added php 8.3 and php 8.4

### CODE REVIEW

File: resources/config/util.yml [hunk 1]
```diff
@@ -20,5 +20,7 @@ services:
               '7.3': '2021-12-06 23:59:59'
               '7.4': '2022-11-28 23:59:59'
               '8.0': '2023-11-26 23:59:59'
-              '8.1': '2024-11-25 23:59:59'
-              '8.2': '2025-12-08 23:59:59'
+              '8.1': '2025-12-31 23:59:59'
+              '8.2': '2027-12-31 23:59:59'
```
Comment (veewee) [inline resources/config/util.yml]: it looks like you swapped 8.2 and 8.3 release date?
At least according to https://www.php.net/supported-versions.php

**Review by veewee — COMMENTED** [review]
Thanks, I've noticed a small misconfiguration.

File: resources/config/util.yml (hunk 1 above)
Comment (jwderoos) [reply resources/config/util.yml]: Did I? Oh, yes looks like I did.

Comment (jwderoos) [reply resources/config/util.yml]: Fixed, sorry about that.

**Pull request merged**